src/components/ContactList.tsx

```diff
@@ -92,8 +92,12 @@ const ContactList = () => {
                     value={searchInput} 
                     onChange={(e) => setSearchInput(e.target.value)}
                 />
-                <button type="button" onClick={handleClick}>Search</button>
-                <Link to="/contacts">See All Contacts</Link>
+                <button id="search-button" type="button" onClick={handleClick}>Search</button>
+            </div>
+            <div id="button-div">
```

Comment:
nit: this is often called a `button group` - just a naming convention I've seen
```suggestion
            <div id="button-group">
```